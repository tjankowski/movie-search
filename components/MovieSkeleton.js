import React from "react";
import Skeleton from "react-loading-skeleton";
import styles from "styles/MovieCard.module.scss";

export default function MovieSkeleton() {
  return (
    <div className={styles.movieCard}>
      <div className={styles.movieCard__posterContainer}>
        <Skeleton width="6rem" height="6rem" />
      </div>
      <div className={styles.movieCard__title}>
        <h3>
          <Skeleton width="8rem" />
        </h3>
        <div>
          <Skeleton width="4rem" />
        </div>
        <div>
          <Skeleton width="4rem" />
        </div>
      </div>
    </div>
  );
}
