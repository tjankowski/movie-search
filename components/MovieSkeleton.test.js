import React from "react";
import renderer from "react-test-renderer";
import MovieSkeleton from "./MovieSkeleton";

describe("MovieSkeleton", () => {
  it("should render", () => {
    const component = renderer.create(<MovieSkeleton />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
