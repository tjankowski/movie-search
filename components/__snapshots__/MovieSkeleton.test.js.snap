// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieSkeleton should render 1`] = `
<div
  className="movieCard"
>
  <div
    className="movieCard__posterContainer"
  >
    <span>
      <span
        className="react-loading-skeleton css-1vmnjpn-skeletonStyles-Skeleton"
        style={
          Object {
            "height": "6rem",
            "width": "6rem",
          }
        }
      >
        ‌
      </span>
    </span>
  </div>
  <div
    className="movieCard__title"
  >
    <h3>
      <span>
        <span
          className="react-loading-skeleton css-1vmnjpn-skeletonStyles-Skeleton"
          style={
            Object {
              "width": "8rem",
            }
          }
        >
          ‌
        </span>
      </span>
    </h3>
    <div>
      <span>
        <span
          className="react-loading-skeleton css-1vmnjpn-skeletonStyles-Skeleton"
          style={
            Object {
              "width": "4rem",
            }
          }
        >
          ‌
        </span>
      </span>
    </div>
    <div>
      <span>
        <span
          className="react-loading-skeleton css-1vmnjpn-skeletonStyles-Skeleton"
          style={
            Object {
              "width": "4rem",
            }
          }
        >
          ‌
        </span>
      </span>
    </div>
  </div>
</div>
`;
